### PR TITLE
Handle nonexist address in URL

### PR DESCRIPTION
**Pull request by tarvos21**

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -95,5 +95,9 @@
     "barnardBlock": "Barnard network latest block number: ",
     "mainnetBlock": "Starcoin Main network will launch at: ",
     "eventsline": "Key Events"
+  },
+  "account": {
+    "address": "Address",
+    "notOnchain": "Does Not Exist On the Starcoin Blockchain"
```
Comment (wk3368) [inline public/locales/en/translation.json:101]: suggest change to:
This address does not exist on this chain.

File: public/locales/zh/translation.json [hunk 2]
```diff
@@ -95,5 +95,9 @@
     "barnardBlock": "Barnard 网络最新区块高度: ",
     "mainnetBlock": "Starcoin 主网上线高度: ",
     "eventsline": "大事记"
+  },
+  "account": {
+    "address": "地址",
+    "notOnchain": "不在 Starcoin 链上"
```
Comment (wk3368) [inline public/locales/zh/translation.json:101]: suggest change to:
该地址在此链上不存在。

File: src/modules/Address/index.tsx [hunk 3]
```diff
@@ -100,7 +110,13 @@ class Index extends PureComponent<IndexProps, IndexState> {
...
 
   render() {
-    const { addressData, balancesData } = this.state;
+    const { addressData, balancesData, accountStatus } = this.state;
+    const hash = this.props.computedMatch.params.hash;
+    if (accountStatus === undefined) {
+      return <Loading />;
+    } else if (accountStatus === 'nonexist') {
+      return <AddressNotFound address={hash} />;
+    }
```
Comment (wk3368) [inline src/modules/Address/index.tsx:119]: why add a redirect ?

you can render following componet directly:
```
<div>
	<Helmet>
	  <title>Address Not Onchain</title>
	</Helmet>
	<Grid className={classes.root} container justify="center">
	  <Grid item xs={12} md={8} lg={4}>
	    <Card className={classes.card}>
	      <Typography variant="h6" className={classes.headline}>
	        {t('account.address')} <strong>{address}</strong> {t('account.notOnchain')}
	      </Typography>
	    </Card>
	  </Grid>
	</Grid>
</div>
```

File: public/locales/en/translation.json (hunk 1 above)
Comment (tarvos21) [reply public/locales/en/translation.json:101]: Rejected.

File: public/locales/zh/translation.json (hunk 2 above)
Comment (tarvos21) [reply public/locales/zh/translation.json:101]: Rejected.

File: src/modules/Address/index.tsx (hunk 3 above)
Comment (tarvos21) [reply src/modules/Address/index.tsx:119]: It's necessary before the balance is returned.

**Review by wk3368 — APPROVED** [review]
LGTM

**Pull request merged**